GS01SUMO/GS01SUMO_R00004_ingest.csv

```diff
@@ -0,0 +1,80 @@
+mi.dataset.driver.pco2w_abc.imodem.pco2w_abc_imodem_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl11/imm/pco2w2/pco2w2*.DAT,GS01SUMO-RII11-02-PCO2WC052,recovered_host,Available,
+#mi.dataset.driver.pco2w_abc.pco2w_abc_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/instruments/dcl11/imm/PCO2W_snC0138/SAMI*.txt,GS01SUMO-RII11-02-PCO2WC052,recovered_inst,Not Expected,No data available
+mi.dataset.driver.pco2w_abc.imodem.pco2w_abc_imodem_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl11/imm/pco2w3/pco2w3*.DAT,GS01SUMO-RII11-02-PCO2WC053,recovered_host,Available,
+#mi.dataset.driver.pco2w_abc.pco2w_abc_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/instruments/dcl11/imm/PCO2W_snC0139/SAMI*.txt,GS01SUMO-RII11-02-PCO2WC053,recovered_inst,Not Expected,No data available
+mi.dataset.driver.phsen_abcdef.imodem.phsen_abcdef_imodem_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl11/imm/phsen1/phsen1*.DAT,GS01SUMO-RII11-02-PHSENE041,recovered_host,Available,
+#mi.dataset.driver.phsen_abcdef.phsen_abcdef_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/instruments/dcl11/imm/PHSEN_snP0101/SAMI*.txt,GS01SUMO-RII11-02-PHSENE041,recovered_inst,Not Expected,No data available
+mi.dataset.driver.phsen_abcdef.imodem.phsen_abcdef_imodem_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl11/imm/phsen2/phsen2*.DAT,GS01SUMO-RII11-02-PHSENE042,recovered_host,Available,
+#mi.dataset.driver.phsen_abcdef.phsen_abcdef_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/instruments/dcl11/imm/PHSEN_snP0179/SAMI*.txt,GS01SUMO-RII11-02-PHSENE042,recovered_inst,Not Expected,No data available
+mi.dataset.driver.cg_cpm_eng.cpm.cg_cpm_eng_cpm_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/cpm1/irid2shore/cpm_status*.txt,GS01SUMO-SBC11-00-CPMENG000,recovered_host,Available,
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl11/syslog/*syslog.log,GS01SUMO-SBD11-00-DCLENG000,recovered_host,Available,
+mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl11/mopak/*.mopak.log,GS01SUMO-SBD11-01-MOPAK0000,recovered_host,Available,
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl11/hyd1/*hyd1.log,GS01SUMO-SBD11-03-HYDGN0000,recovered_host,Available,
+mi.dataset.driver.dosta_abcdjm.dcl.dosta_abcdjm_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl11/dosta1/*.dosta1.log,GS01SUMO-SBD11-04-DOSTAD000,recovered_host,Available,
+mi.dataset.driver.spkir_abj.dcl.spkir_abj_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl11/spkir1/*.spkir1.log,GS01SUMO-SBD11-05-SPKIRB000,recovered_host,Available,
+mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_recovered_driver,/omc_data/whoi/OMC/GS01SUMO/R00004/cg_data/dcl11/metbk1/*.metbk1.log,GS01SUMO-SBD11-06-METBKA000,recovered_host,Available,

```

Comment:
Do we not have a driver for METBK recovered instrument?  We download data from both the Logger and the individual modules.

Reply:
Correct, there is only a driver/parser for metbk dcl and we've only ingested the logger data.